microcosm/tracing.py

```diff
@@ -0,0 +1,31 @@
+from jaeger_client import Config
+
+from microcosm.api import binding, defaults, typed
+
+
+TRACE_ID = "uber-trace-id"
```

Comment:
uber?

Reply:
That's the name of the header they use. I'll add a comment.

Reply:
Turns out I can delete this, it's also in the constants for the library.